Runtime/Dividat/PluginWrapper/Hardware.cs

```diff
@@ -68,9 +72,8 @@ public static void ActivateMotor(MotorPattern pattern)
         public static void Wire()
         {
 #if UNITY_WEBGL && !UNITY_EDITOR
-            Register(OnStep, OnRelease, OnSensoState);
 #elif UNITY_EDITOR
-            WebsocketAvatar.Instance.Register(OnStep, OnRelease, OnSensoState);
+            WebsocketAvatar.Instance.Register();
```

Comment:
@sutee9 This `Register` method seems a bit pointless now, because it is empty and only needed for the side-effect of instantiating the singleton.

Should we move the instantiation to `Register` explicitly? Or do you have another suggestion for how to ensure instantiation happens if a computed property is used?